helix-term/src/commands.rs

```diff
@@ -2485,6 +2485,10 @@ mod cmd {
         args: &[&str],
         _event: PromptEvent,
     ) -> anyhow::Result<()> {
+        if args.len() == 0 {
+            return Ok(());
```

Comment:
We should return an error here, rather than failing silently.
```suggestion
            bail!("Line number required");
```

Reply:
Thanks, that would be better than silencing the false path. :)

Reply:
@Omnikar Fixed. :)